github/migrations_user.go

```diff
@@ -97,8 +97,12 @@ func (s *MigrationService) StartUserMigration(ctx context.Context, repos []strin
 // ListUserMigrations lists the most recent migrations.
 //
 // GitHub API docs: https://docs.github.com/en/rest/migrations/users#list-user-migrations
-func (s *MigrationService) ListUserMigrations(ctx context.Context) ([]*UserMigration, *Response, error) {
+func (s *MigrationService) ListUserMigrations(ctx context.Context, opts *ListOptions) ([]*UserMigration, *Response, error) {
 	u := "user/migrations"
+	u, err := addOptions(u, opts)
+	if err != nil {
+		return nil, nil, err
+	}
```

Comment:
It would be nice if this case got tested.

Reply:
@gmlewis @raynigon I am actually not sure how I can test this path..The `ListOptions{}` takes in `Page` and `PerPage`. Is there an example you can point me to?

Reply:
Ah, right... with the fixed URL, I'm not sure this can be tested, so let's leave it as-is.  Thanks, @amitsaha .